assets/images/logos/README.md

```diff
@@ -0,0 +1,10 @@
+# Logo
+
+The logo was originally designed by [Afolabi Rahmot](https://github.com/rahmot) using CorelDraw
```

Comment:
If there an SVG for the logo that you can add there? in case we need to export it with different background or size

Reply:
> If there an SVG for the logo that you can add there? in case we need to export it with different background or size

Yes! I have added the SVG in the folder